ui/datalist.jsx

```diff
@@ -0,0 +1,13 @@
+import React from 'react';
+
+const DataList = ({data}) => {
```

Comment:
Add prefix and export:

```suggestion
export const SoDataList = ({data}) => {
```